src/System.IO.Pipes/src/System/IO/Pipes/AnonymousPipeServerStreamAcl.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace System.IO.Pipes
+{
+    public static class AnonymousPipeServerStreamAcl
+    {
+        public static AnonymousPipeServerStream Create(
+            PipeDirection direction,
+            HandleInheritability inheritability,
+            int bufferSize,
+            PipeSecurity pipeSecurity)
+        {
+            return null;
```

Comment:
Same as on the other PR; I'm confused by these `return null`s.  What's the purpose of this PR?